servant/src/Servant/API/MultiVerb.hs

```diff
@@ -0,0 +1,486 @@
+{-# LANGUAGE ApplicativeDo #-}
+{-# LANGUAGE EmptyCase #-}
+
+-- | MultiVerb is a part of the type-level eDSL that allows you to express complex routes
+-- while retaining a high level of precision with good ergonomics.
+
+module Servant.API.MultiVerb
```

Comment:
oh, another thought: should we deprecate `UVerb`?  if not: should we add a comment to the haddocks that `MultiVerb` is strictly more powerful and should be considered as an alternative?

Reply:
We'll deprecate UVerb one day, and yes it should be labelled as an alternative.